src/app/modules/gb-cohort-selection-module/constraint-components/gb-constraint/gb-constraint.component.ts

```diff
@@ -98,7 +98,7 @@ export class GbConstraintComponent implements OnInit {
   get observationBoxMessage(): string {
     let parentDimension = this.constraint.parentDimension;
     if (this.constraint.negated) {
-      return `for the ${parentDimension} there is NO observation:`;
+      return `for the ${parentDimension} there is an observation other than observations:`;
```

Comment:
one related cypress test: https://github.com/thehyve/glowing-bear/blob/dev/cypress/support/step_definitions/constraint-creation.js#L90

Reply:
Thanks!

Reply:
I updated it.